### PR TITLE
ES: Localize term 'distributed apps' apps in Spanish.

### LINKED ISSUE #600: [Spanish] localize 'Distributed Apps'
Localize the `content/en/distributed_apps.md` document to Spanish

### PR DESCRIPTION
**Pull request by 92nqb**
### Describe your changes

Localization of distributed-apps.md

### Related issue number or link (ex: `resolves #issue-number`)

It closes #600 

### Checklist before opening this PR (put `x` in the checkboxes)
- [X] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [X] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/es/distributed-apps.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+title: Aplicaciones Distribuidas
+status: Completed
+category: concept
+tags: ["architecture", "", ""]
+---
+
+## ¿Qué es?
+
+Una aplicación distribuida es una aplicación en la que la funcionalidad se divide en múltiples partes independientes más pequeñas. 
+Las aplicaciones distribuidas suelen estar formadas por componentes individuales llamados [microservicios](/microservices/)
```
Comment (electrocucaracha) [inline content/es/distributed-apps.md:11]:
```suggestion
Las aplicaciones distribuidas suelen estar formadas por componentes individuales llamados [microservicios](/es/microservices/)
```

File: content/es/distributed-apps.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+que se ocupan de diferentes responsabilidades dentro de una aplicación más extensa. 
+En un entorno de nube nativa, los componentes individuales suelen ejecutarse como [contenedores](/container/) en un [clúster](/cluster/).
```
Comment (electrocucaracha) [inline content/es/distributed-apps.md:13]:
```suggestion
En un entorno nativo para la nube, los componentes individuales suelen ejecutarse como [contenedores](/container/) en un [clúster](/es/cluster/).
```

File: content/es/distributed-apps.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+
+## Problema que aborda
+
+Una aplicación que se ejecuta en un único ordenador representa un único punto de fallo: si ese ordenador falla, la aplicación deja de estar disponible.
```
Comment (electrocucaracha) [inline content/es/distributed-apps.md:18]: https://es.wikipedia.org/wiki/Punto_%C3%BAnico_de_fallo
```suggestion
Una aplicación que se ejecuta en un único ordenador representa un punto único de fallo: si ese ordenador falla, la aplicación deja de estar disponible. 
```

File: content/es/distributed-apps.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+Una aplicación distribuida es una aplicación en la que la funcionalidad se divide en múltiples partes independientes más pequeñas. 
+Las aplicaciones distribuidas suelen estar formadas por componentes individuales llamados [microservicios](/microservices/)
+que se ocupan de diferentes responsabilidades dentro de una aplicación más extensa. 
+En un entorno de nube nativa, los componentes individuales suelen ejecutarse como [contenedores](/container/) en un [clúster](/cluster/).
+
+
+## Problema que aborda
+
+Una aplicación que se ejecuta en un único ordenador representa un único punto de fallo: si ese ordenador falla, la aplicación deja de estar disponible. 
+Las aplicaciones distribuidas a menudo se contraponen a las [aplicaciones monolíticas](/monolithic-apps/).
```
Comment (electrocucaracha) [inline content/es/distributed-apps.md:19]:
```suggestion
Las aplicaciones distribuidas a menudo se contraponen a las [aplicaciones monolíticas](/es/monolithic-apps/). 
```

File: content/es/distributed-apps.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+que se ocupan de diferentes responsabilidades dentro de una aplicación más extensa. 
+En un entorno de nube nativa, los componentes individuales suelen ejecutarse como [contenedores](/container/) en un [clúster](/cluster/).
+
+
+## Problema que aborda
+
+Una aplicación que se ejecuta en un único ordenador representa un único punto de fallo: si ese ordenador falla, la aplicación deja de estar disponible. 
+Las aplicaciones distribuidas a menudo se contraponen a las [aplicaciones monolíticas](/monolithic-apps/). 
+Una aplicación monolítica puede ser más difícil de escalar, ya que los distintos componentes no pueden escalarse de forma independiente. 
+También pueden convertirse en un lastre para la velocidad de los desarrolladores a medida que crecen
```
Comment (electrocucaracha) [inline content/es/distributed-apps.md:21]:
```suggestion
También pueden convertirse en un lastre en la velocidad de desarrollo a medida que crecen 
```

File: content/es/distributed-apps.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+porque más desarrolladores tienen que trabajar en una base de código compartida que no necesariamente tiene límites bien definidos.
+
+
+## ¿Cómo ayuda?
+
+Al dividir una aplicación en diferentes partes y ejecutarlas en muchos lugares, el sistema global puede tolerar más fallos. 
+También permite a una aplicación aprovechar características de escalado no disponibles para aplicaciones de una única instancia, 
+es decir, permite aprovechar la capacidad de [escalar horizontalmente](/horizontal-scaling/). 
+Sin embargo, esto tiene un precio: una mayor complejidad y sobrecarga operativa. 
+- ahora estás ejecutando muchos componentes de aplicación en lugar de una única aplicación.
```
Comment (electrocucaracha) [inline content/es/distributed-apps.md:31]:
```suggestion
- ahora estás ejecutando muchos componentes en lugar de una aplicación única.
```

**Review by electrocucaracha — CHANGES_REQUESTED** [review]

**Comment by 92nqb** [comment]
@electrocucaracha, thanks for your feedback. 
I've accepted your suggestions :)

**Review by krol3 — APPROVED** [review]
LGTM

File: content/es/distributed-apps.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+porque más desarrolladores tienen que trabajar en una base de código compartida que no necesariamente tiene límites bien definidos.
+
+
+## ¿Cómo ayuda?
+
+Al dividir una aplicación en diferentes partes y ejecutarlas en muchos lugares, el sistema global puede tolerar más fallos. 
+También permite a una aplicación aprovechar características de escalado no disponibles para aplicaciones de una única instancia, 
+es decir, permite aprovechar la capacidad de [escalar horizontalmente](/horizontal-scaling/). 
+Sin embargo, esto tiene un precio: una mayor complejidad y sobrecarga operativa. 
+- ahora estás ejecutando muchos componentes en lugar de una aplicación única.
```
Comment (electrocucaracha) [inline content/es/distributed-apps.md:31]: Parece ser que en inglés usan un carácter diferente, por lo que el rendering se comporta distinto
```suggestion
— ahora estás ejecutando muchos componentes en lugar de una aplicación única.
```

**Review by electrocucaracha — APPROVED** [review]

**Pull request merged**